GW2EIEvtcParser/ParserHelpers/ArcDPSEnums.cs

```diff
@@ -1206,6 +1206,13 @@ public enum MinionID : int
             FireElemental = 6524,
             IceElemental = 6525,
             //
+            SneakGyro = 15012,
```

Comment:
Any intent to use those somewhere down the line? Or just putting them down for reference?

Reply:
It's just for historical value, I will add the icons to ParserIcons later on. I'm waiting on a friend to make them so they can show up in the combat replay (does that even exist for 2018-2019 logs?)

Reply:
2018 is very unlikely as movement data for non players/non boss ids were introduced much later, 2019 should be fine~ish depending on the date. Would need to be checked.